media/css/mozorg/contribute/friends.less

```diff
@@ -28,188 +47,372 @@
             padding: 0 20px;
         }
     }
+
     #colophon {
         margin-top: 0;
     }
+
     h1,
     h2,
     h3 {
         color: #fff;
         font-weight: 700;
     }
+
     h2 {
         .font-size(2.5rem);
     }
-    h3 {
-        .font-size(1.2rem);
-        text-transform: uppercase;
 
-        @media only screen and (max-width: 790px) {
-            .font-size(1rem);
-            line-height: 1.5;
-        }
-    }
     .ff-logo {
         padding-top: 25px;
     }
+
     .ff-header-content {
         color: #fff;
         margin: 0 auto;
-        padding: 95px 0 160px 0;
-        width: 100%;
+        padding: 55px 0 80px 0;
         text-align: center;
+        width: 100%;
 
         @media only screen and (max-width: @breakMobileLandscape) {
             padding-bottom: 80px;
         }
 
         h1 {
-            .font-size(4rem);
+            .font-size(64px);
+            margin-bottom: @baseLine * 2;
             text-shadow: 1px 1px 2px #333;
 
             @media only screen and (max-width: 790px) {
-                .font-size(3rem);
+                .font-size(48px);
+            }
+
+            @media only screen and (max-width: @breakMobileLandscape) {
+                .font-size(40px)
             }
         }
         p {
-            margin: 0 auto;
-            max-width: 610px;
             .font-size(1.4rem);
+            margin: @baseLine auto 0;
+            max-width: 700px;
             text-shadow: 1px 1px 2px #333;
 
             @media only screen and (max-width: 790px) {
+                .font-size(18px);
                 max-width: 500px;
-                .font-size(1.2rem);
             }
-        }
-    }
-    .ff-join-now {
-        display: block;
-        background-color: #f26c24;
-        color: #fff;
-        margin: 70px auto 0 auto;
-        padding: 10px 30px;
-        border: 0;
-        border-radius: 6px;
-        width: 200px;
-        .font-size(1.2rem);
 
-        @media only screen and (max-width: @breakMobileLandscape) {
-            margin-top: 35px;
+            @media only screen and (max-width: @breakMobileLandscape) {
+                .font-size(@largeFontSize);
+            }
         }
 
-        &:hover {
-            text-decoration: none;
+        .ff-join-now {
+            margin: 70px auto 0 auto;
+            transition: opacity 0.3s ease-in 0.1s;
```

Comment:
Will do. On that note, how much longer you think we should keep using the mixin? Older Android looks like the major factor, but it has supported the standard syntax for three versions now, and transitioning *is* a progressive enhancement...